libctru/source/services/fs.c

```diff
@@ -9,20 +9,30 @@
 #include <3ds/env.h>
 #include "../internal.h"
 
+#define FS_MAX_ARCHIVE_HANDLES 32
```

Comment:
I'm perfectly fine with editing in a compromise between the two if you can give me a specific number.